docs/changelog.rst

```diff
@@ -41,6 +41,7 @@ In preparation for HD Views developments (PR #3298)
 - Check pictogram exist on categories during generation of pdfs
 - Fix ApidaeParsers does not update every time
 - Add fixtures licenses initial install
+- Replace trek's description relative url with absolute url on api v2 (#3321)
```

Comment:
- proposed phrasing: "Replace image's relative URLs with absolute URLs in API v2 trek descriptions"

Reply:
Could you add a link to the issue for the "#3321", thx :pray: